### PR TITLE
Arcbox refresh - updated SAS and SQL VM VHD

### PR DESCRIPTION
**Pull request by chintalavr**
This PR includes changes to support
- New SAS token to address Arc SQL Jumpstart scenarios
- Using VHD from ArcBox to avoid duplicate VHD maintenance.
- Updated documentation to provide Portal deployment option for Defender for SQL Jumpstart scenario.

### CODE REVIEW

**Review by dkirby-ms — CHANGES_REQUESTED** [review]
Please remove Az CLI pindown. Otherwise looks good.

File: azure_jumpstart_arcbox/artifacts/Bootstrap.ps1 [hunk 1]
```diff
@@ -142,6 +142,8 @@ foreach ($app in $appsToInstall) {
     
 }
 
+choco install azure-cli --version=2.49.0 -y
```
Comment (sebassem) [inline azure_jumpstart_arcbox/artifacts/Bootstrap.ps1:145]: No need to do Az CLI pindown

**Review by sebassem — CHANGES_REQUESTED** [review]

File: azure_jumpstart_arcbox/artifacts/Bootstrap.ps1 (hunk 1 above)
Comment (chintalavr) [reply azure_jumpstart_arcbox/artifacts/Bootstrap.ps1]: Reverted changes and good to go.

**Comment by dkirby-ms** [comment]
Should this be merged before Seif's pindown PR?

**Comment by sebassem** [comment]
> Should this be merged before Seif's pindown PR?

I think so, lets merge this first then I will review if I need to make any changes to my PR

**Review by dkirby-ms — APPROVED** [review]

**Review by sebassem — APPROVED** [review]

**Pull request merged**